static/js/movie.js

```diff
@@ -226,3 +226,56 @@ function renderNextPage(data) {
   const previousBtn = document.querySelector(".previous-btn");
   previousBtn.addEventListener("click", previousReviewPage);
 }
+
+// Send a review for the movie \\
+let rate = document.querySelector("#rate");
+let comment = document.querySelector("#addComment");
+let authorName = document.querySelector("#addName");
+let reviewBox = document.querySelector(".addReview");
+
+// Change temporary content of adding review box into Thank you text 
+function changeContent() {
+  var form = document.getElementById("form")
+  var originalContent = reviewBox.innerHTML;
+  reviewBox.innerHTML = "Tack för din recension!";
+  setTimeout(function() {
+    reviewBox.innerHTML = originalContent;
+  }, 10000) };
```

Comment:
Nice feature!

The code formatting makes it a little bit difficult to read, however.